common/src/main/java/org/wso2/testgrid/common/TestGridConstants.java

```diff
@@ -47,5 +47,6 @@
 
     public static final String WUM_USERNAME_PROPERTY = "WUMUsername";
     public static final String WUM_PASSWORD_PROPERTY = "WUMPassword";
+    public static final String CUSTOM_USER_DATA = "CustomUserData";
```

Comment:
Testgrid core is independent of the cloud providers. Seems this shud be added to AWSProvider.

Reply:
Fixed with commit https://github.com/wso2/testgrid/pull/747/commits/8dffa11a3baf2ab1b238094e4301c57e85a04be7